### ISSUE TITLE
Deck of 🂱

### ISSUE THREAD

**Issue by dvorka** [dvorka/mindforger#1299]
Feature which turns knowledge managed by MindForger to ultimate learning tool

**Comment by VictorGrycuk** [comment]
Is this related to Anki cards? It would be awesome to be able to create anki cards using my notebooks/notes.
Extra points if I can import them to the Anki app, so I can sync it with my cellphone.

**Comment by dvorka** [comment]
@VictorGrycuk Yes, exactly - this is the idea ;-) Thank you for Anki integration suggestion!

**Comment by hannesfrank** [comment]
I love Spaced Repetition (Anki user), so thanks for thinking about this feature!

I want to mention [RemNote](https://www.remnote.io/) for reference. It is an outliner similar to RoamResearch which transforms every bullet point with a special marker into a card (front: before marker, back: after marker). Maybe you can borrow some concepts from there, if you don't already have something in mind.
